doc/tutorial/tutorial_drawing_smartview.rst

```diff
@@ -0,0 +1,757 @@
+
+c) Face, as called as node faces, are small pieces of extra graphical information that can be linked to nodes (text labels, images, graphs, etc.). Several types of node faces are provided by the previous ETE3 module, ranging from simple text (TextFace) and geometric shapes (CircleFace), to molecular sequence representations (SequenceFace), etc. These faces are upgraded in ETE4 to adapt the large tree drawing engine.
+
+d) TreeLayout, is a class which defines a foundational layout for trees to set specific styles for both the entire tree and individual nodes, acting as a pre-drawing hooking framework. When a tree is about to be drawn, the above elements such as TreeStyle, NodeStyle, Face of nodes can be then set up and modified on the fly and returned to the drawer engine. Hence TreeLayout class can be understood as a suite of rules tree’s basic setting and how different nodes should be drawn. 
+
+Scheme of fundamental components in ETE4's programmable tree drawing engine
+
+.. image:: https://github.com/dengzq1234/ete4_gallery/blob/master/smartview/fundamental_ete4.jpg?raw=true
+   :alt: alternative text
+   :align: center
+
+Explore interactive isualization of trees 
+-----------------------------------------
+
+ETE's tree drawing engine is fully integrated with a built-in
+graphical user interface (GUI) which allows to explore and manipulate
+node's properties and tree topology. To start the visualization of a
+node (tree or subtree), you can simply call the :func:`explore
+<ete4.Tree.explore>` method.
+
+One of the advantages of this visualization is that you can use it to
+interrupt a given program/analysis, explore the tree, manipulate it,
+and continue with the execution. Note that **changes made using the
+GUI will be kept after quiting the GUI**. This feature is specially
+useful during python sessions, and it can be utilized in various environments 
+by modifying argument *keep_server*, including standalone scripts and interactive 
+sessions such as IPython or Jupyter Notebooks. Below are examples demonstrating 
+the method's usage in each context.
+
+Standalone scripts
+~~~~~~~~~~~~~~~~~~
+When running a standalone script, *keep_server* should be set as *True* to keep 
+the server running.
+
+::
+
+  from ete4 import Tree
+  t = Tree('((a,b),c);')
+  t.explore(keep_server=True)
+
+Source code can be found in in ETE4 here: `explore_standalone.py example <https://github.com/dengzq1234/ete4_gallery/blob/master/smartview/explore_standalone.py>`_.
+
+Interactive sessions
+~~~~~~~~~~~~~~~~~~~~
+When running in interactive sessions such as IPython or Jupyter Notebooks, *keep_server* 
+should be set as *False*
```

Comment:
Actually, better if they don't set it at all (why bother the user with options they don't need).